src/WebUI/ImageDetails/ImageDetails.scss

```diff
@@ -22,18 +22,17 @@
 }
 // end of above common css copied
 
-.image-details-content {
-    .image-details-header {
-        padding-left: 0;
-        margin-left: 0;
+.image-details-content .image-details-header .image-details-back-button {
+    cursor: pointer;
+}
 
-        .image-details-back-button {
-            cursor: pointer;
-        }
-    }
+// todo :: once summary card is available we should remove this
+.image-details-key .image-o-k-col-content,
+.image-details-value .image-o-v-col-content {
```

Comment:
What does 'o' stand for in these class names? k & v I understand are for key & value respectively.

Reply:
'o' is overview, just wanted to keep the strings smaller.